### DISCUSSION TITLE
Removal of semicolons

### DISCUSSION THREAD

**Discussion by arthuro555** [jeremyckahn/shifty#127] [Q&A] [answered]
I noticed this commit from a few months ago: https://github.com/jeremyckahn/shifty/commit/8c33d79fbf3fa05ab73c208f79922c48c3eb2955
I wondered, why the removal of semicolons? Is it something performance related, a stylistic preference, or because making sure they were there was too much of a hassle?

**Comment by jeremyckahn** [comment, marked as answer]
Hi @arthuro555! It is strictly a stylistic preference. There’s no performance difference, and even if there was, the semicolons get re-inserted into the build artifacts by the build tool.

I just personally prefer terser code and less typing, and [Prettier made it very easy to make the change](https://github.com/jeremyckahn/shifty/blob/4ded21e919ee5bdc99635cc502caaa9fe723499b/.prettierrc#L2). 🙂
